backend/lib/spree/backend.rb

```diff
@@ -7,6 +7,7 @@
 require 'glyphicons'
 require 'popper_js'
 require 'inline_svg'
+require 'active_storage_validations'
```

Comment:
@MatthewKennedy, for now, we can add this gem, in Rails 6.1 this will be part of the Rails core